doc/contexts.md

```diff
@@ -38,4 +38,11 @@ This can be done easily by extending the `SparkContextFactory` trait, like `SQLC
 
 ## Jars
 
-If you wish to use the `SQLContext` or `HiveContext`, be sure to pull down the job-server-extras package.
\ No newline at end of file
+If you wish to use the `SQLContext` or `HiveContext`, be sure to pull down the job-server-extras package.
+
+# StreamingContext
+job-server-extras provides a context to run Spark Streaming jobs, there are a couple of configurations you can change in job-server's .conf file
+streaming.batch_interval: the streaming batch in millis
```

Comment:
We might need to reformat this for Markdown using lists.  Can fix it later after merge.